data/data.json

```diff
@@ -7923,10 +7923,10 @@
   },
   "v1.19.16-rancher1-3": {
```

Comment:
You're only updating the latest mirror version of kubernetes right, hyperkube update 3? I want to make sure I understand why rancher1-1 and 1-2 are left out.

Reply:
@annablender yep, that's correct - we're only updating the latest for each minor version, so just rancher1-3 for v1.19.16.